runtime/src/builtin/singletons.rs

```diff
@@ -1,22 +1,31 @@
-// Copyright 2019-2022 ChainSafe Systems
 // SPDX-License-Identifier: Apache-2.0, MIT
 
 use fvm_shared::address::Address;
 use fvm_shared::ActorID;
 
-lazy_static! {
-    pub static ref SYSTEM_ACTOR_ADDR: Address         = Address::new_id(0);
-    pub static ref INIT_ACTOR_ADDR: Address           = Address::new_id(1);
-    pub static ref REWARD_ACTOR_ADDR: Address         = Address::new_id(2);
-    pub static ref CRON_ACTOR_ADDR: Address           = Address::new_id(3);
-    pub static ref STORAGE_POWER_ACTOR_ADDR: Address  = Address::new_id(4);
-    pub static ref STORAGE_MARKET_ACTOR_ADDR: Address = Address::new_id(5);
-    pub static ref VERIFIED_REGISTRY_ACTOR_ADDR: Address = Address::new_id(6);
+use paste::paste;
 
-    pub static ref CHAOS_ACTOR_ADDR: Address    = Address::new_id(98);
+macro_rules! define_singletons {
+    ($($name:ident = $id:literal,)*) => {
+        $(
+            paste! {
```

Comment:
This won't have any affect at runtime, it just allows us to concatenate identifiers at compile time.